oozie/__init__.py

```diff
@@ -0,0 +1,5 @@
+# Copyright (c) 2017 "Shopify inc." All rights reserved.
```

Comment:
I'm not sure about naming this package `oozie`. This is not `oozie`, this is a library that interoperates with Oozie.

The first example that came to mind as PyWebHDFS, which uses the package name `pywebhdfs`. It's a library that interoperates with WebHDFS.